alodataset/kitti_stereo_flow_sflow2015.py

```diff
@@ -0,0 +1,385 @@
+import os
+import cv2
+import torch
+import numpy as np
+from typing import Dict, List, Tuple
+
+from alodataset import BaseDataset, Split, SplitMixin
+from aloscene import Frame, Disparity, Mask, Flow, SceneFlow
+from aloscene.camera_calib import CameraIntrinsic, CameraExtrinsic
+
+
+class KittiStereoFlowSFlow2015(BaseDataset, SplitMixin):
+    SPLIT_FOLDERS = {Split.TRAIN: "training", Split.TEST: "testing"}
+
+    def __init__(
+        self,
+        name="kitti_stereo2015",
+        sequence_start=10,
+        sequence_end=11,
+        grayscale=True,
+        load: list = [
+            "right_frame",
+            "disp_noc",
+            "disp_occ",
+            "flow_occ",
+            "flow_noc",
+            "scene_flow",
+            "obj_map",
+        ],
+        **kwargs,
+    ):
+        super().__init__(name=name, **kwargs)
+        self.sequence_start = sequence_start
+        self.sequence_end = sequence_end
+        self.grayscale = grayscale
```

Comment:
The 2015 dataset is created from RGB images. We do not need to provide grayscale in this dataset class.

Reply:
The grayscale images are provided as an extension. So I just change the parameter as False by default and verify no ground truth is loaded when grayscale is True.